widget/index.html

```diff
@@ -30,74 +33,84 @@
         }
     </style>
 </head>
+
 <body>
-<div class="text-center link-verified">
-    <div class="col-md-12">
-        <span class="glyphicon glyphicon-ok" aria-hidden="true"></span>
-        <p>
-            We were able to verify the link you provided. The link will only appear on your mobile device.
-        </p>
+    <div class="text-center link-verified">
+        <div class="col-md-12">
+            <span class="glyphicon glyphicon-ok" aria-hidden="true"></span>
+            <p>
+                We were able to verify the link you provided. The link will only appear on your mobile device.
+            </p>
+        </div>
     </div>
-</div>
-<script>
-    buildfire.spinner.show();
+    <script>
+        buildfire.spinner.show();
 
-    function dataLoadedHandler(result) {
-        var content = null;
-        if (result && result.data && !angular.equals({}, result.data)) {
-            if (result.data.content) {
-                content = result.data.content;
-            } else if(!result.id && !result.data.content) {
+        function dataLoadedHandler(result) {
+            var content = null;
+            if (result && result.data && !angular.equals({}, result.data)) {
+                if (result.data.content) {
+                    content = result.data.content;
+                } else if (!result.id && !result.data.content) {
+                    content = {
+                        facebookId: "https://www.facebook.com/zuck"
+                    }
+                }
+            } else {
                 content = {
                     facebookId: "https://www.facebook.com/zuck"
-                }
+                };
             }
-        } else {
-            content = {
-                facebookId: "https://www.facebook.com/zuck"
-            };
+            buildfire.spinner.hide();
+            buildfire.getContext(function (err, context) {
+                if (context.device.platform != "web") {
+                    buildfire.navigation.openWindow(content.facebookId, "_blank");
```

Comment:
Why use action items? Why not start with _system ? _blank should never work

Reply:
I only edited HTML5 version, where the issue was with PWAs. Also, I was testing all options, _blank, _system, and action items, and with all of them I get popup window on iOS PWA.